spoc/snipping/snipping_strategies.py

```diff
@@ -0,0 +1,240 @@
+"""Snipping strategies for Snipper that implement specific snipping functionality"""
+from abc import ABC, abstractmethod
+from functools import partial
+from enum import Enum
+from typing import Union
+from multiprocess.pool import ThreadPool
```

Comment:
1. `multiprocess` is absent from any dependencies.
2. Why `multiprocess` instead of `multiprocessing`?

Reply:
Good catch! Will fix it!
I think this was just my VSCode autocompleting imports ;) I have multiprocess installed because cooler/cooltools depends on it. It is a fork of multiprocessing with more advanced serialization for interprocess communication.